src/site/content/en/metrics/fcp/index.md

```diff
@@ -0,0 +1,132 @@
+
+## How to measure FCP
+
+FCP can be measured [in the lab](/user-centric-metrics/#in-the-lab) or [in the
+field](/user-centric-metrics/#in-the-field), and it's available in the following
+tools:
+
+### Lab tools
+
+- [Lighthouse](https://developers.google.com/web/tools/lighthouse/)
+- [WebPageTest](https://www.webpagetest.org/)
+- [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/)
+- [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/)
+
+### Field tools
+
+- [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/)
+- [Chrome User Experience
+  Report](https://developers.google.com/web/tools/chrome-user-experience-report)
+- [Firebase Performance
+  Monitoring](https://firebase.google.com/docs/perf-mon/get-started-web) (beta)
+
+### Measure FCP in JavaScript
+
+You can measure FCP in JavaScript using the [Paint Timing
+API](https://w3c.github.io/paint-timing/). The following example shows how to
+create a
+[`PerformanceObserver`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver)
+that listens for paint timing entries and logs the `first-contentful-paint` time
+to the console:
+
+```js
+// Create the Performance Observer instance.
+const observer = new PerformanceObserver((list) => {
+  for (const entry of list.getEntriesByName('first-contentful-paint')) {
+    // Log the value of FCP to the console.
+    console.log('FCP:', entry.startTime);
+  }
+});
+
+// Start observing paint entry types.
+observer.observe({
+  type: 'paint',
+  buffered: true,
+});
+```
+
+In your own code, you'd likely replace the `console.log()` with code that sends
+the FCP value to your analytics service.
+
+## What is a good FCP?
+
+[RAIL](https://developers.google.com/web/fundamentals/performance/rail)
+guidelines suggest that after 1000 milliseconds (ms), users may lose focus if
+nothing happens after performing a task. Since First Contentful Paint is the
+first indication users will get that something is happening, it's best if the
+majority of page loads report FCP is less than a second (1000 ms).
+
+**For field measurement:** Page Speed Insights will
+[report](https://developers.google.com/speed/docs/insights/v5/about#distribution)
+FCP as "fast" for values 1 second or less, "moderate" for values between 1 and 3
+seconds, and "slow" for any values above 3 seconds.
+
+**For lab measurement:** Lighthouse will give a perfect score (100) to an FCP
+value 1 second or less, and the score will go down as FCP gets slower. Refer to
+the [Lighthouse scoring
+guide](https://developers.google.com/web/tools/lighthouse/v3/scoring#perf) for
+more details.
+
+## How to improve FCP
+
+To learn how to improve FCP for a specific site, you can run a Lighthouse
+performance audit and pay attention to any specific
+[opportunities](/lighthouse-performance#opportunities) or
+[diagnostics](/lighthouse-performance#diagnostics) the audit suggests.
```

Comment:
These two take me to the same link, is it because this is staging or are the links incorrect?

Reply:
Thanks! Nice catch. Fixed.